apps/frontend/src/lib/imports/imports.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  AccessToken,
+  ChipIssuer,
+  DataOption,
+  errorToString, OAuthAppDetails
+} from "@types";
+import { toast } from "sonner";
+import { User } from "@/lib/storage/types";
+import { fetchImportedData } from "./fetch";
+import { saveImportedData } from "./save";
+import { OAUTH_APP_DETAILS } from "@/config";
+import { storage } from "@/lib/storage";
+import { getOAuthAccessToken } from "@/lib/oauth";
+
+export async function fetchAndSaveImportedData(
+  authToken: string,
+  user: User,
+  chipIssuer: ChipIssuer,
```

Comment:
For now maybe we can just remove chipIssuer and fetch this globally. Chip issuers can answer the question of "who does your data get shared with" post-Devcon, but for now we can just assume that data imports are global

Reply:
nvm, i see saving is for updating leaderboard, hmm.

Reply:
In the next PR I generalize it so it can be supplied all the communities that need leaderboard entries.